clang/lib/Format/UnwrappedLineFormatter.cpp

```diff
@@ -1596,7 +1596,7 @@ static auto computeNewlines(const AnnotatedLine &Line,
     if (Line.startsWith(TT_NamespaceRBrace)) {
       if (Style.WrapNamespaceBodyWithEmptyLines == FormatStyle::WNBWELS_Never)
         Newlines = 1;
-      else if (!PreviousLine->startsWith(TT_NamespaceRBrace))
+      else if (PreviousLine && !PreviousLine->startsWith(TT_NamespaceRBrace))
```

Comment:
As owen said, we need a test case.

If `Line` starts with a closing namespace, I don't see how there could not be a `PreviousLine`.

Reply:
Thank you @HazardyKnusperkeks and @owenca - I've decided to revert this at present.